src/control/client/storage_query.go

```diff
@@ -65,3 +65,29 @@ func (c *connList) SmdListDevs(req *mgmtpb.SmdDevReq) ResultSmdMap {
 
 	return results
 }
+
+// DevStateQuery will print the state of the given device UUID
+func (c *connList) DevStateQuery(req *mgmtpb.DevStateReq) ResultStateMap {
+	results := make(ResultStateMap)
+	mc := c.controllers[0] // connect to first AP only for now
```

Comment:
This shouldn't be hard-coded here. The chooseServiceLeader() function should be used instead for future-proofing.

Reply:
I didn't notice that this functionality got added! thanks for pointing out :)